locales-en-US.xml

```diff
@@ -301,6 +525,19 @@
     </term>
 
     <!-- VERB ROLE FORMS -->
+    <term name="chair" form="verb"></term> <!-- ??? -->
+    <term name="compiler" form="verb">compiled by</term>
+    <term name="contributor" form="verb">contributions by</term>
+    <term name="curator" form="verb">curated by</term>
+    <term name="executive-producer" form="verb">produced by</term>
+    <term name="guest" form="verb"></term> <!-- ??? -->
```

Comment:
```suggestion
    <term name="guest" form="verb">with guest</term>
```

@adam3smith What do you think? Or just `with`?

Reply:
I think this works, yes